### PR TITLE
Schedule daily backups

### PR DESCRIPTION
**Pull request by JReissmueller**
Added api request to make Vultr server backups daily when backups are enabled.

### CODE REVIEW

File: vultr.php [hunk 1]
```diff
@@ -1395,6 +1402,11 @@ public function editService($package, $service, array $vars = null, $parent_pack
                     ];
                     $this->log('api.vultr.com|backup_enable', serialize($params), 'input', true);
                     $this->parseResponse($vultr_api->backupEnable($params));
+
```
Comment (tysonphillips) [inline vultr.php:1415]: Line 1397 checks if the backup option is enabled and then makes two calls to enable backups and set the backup schedule to daily. What if the config option was already enabled and we're just re-saving the service with existing settings? Will this then make two API requests again?

Comment (JReissmueller) [reply vultr.php:1415]: Yes

Comment (tysonphillips) [reply vultr.php:1415]: That sounds unnecessary. Can we just check if the value differed and then make the calls only then?

Comment (JReissmueller) [reply vultr.php:1415]: Yes

Comment (JReissmueller) [reply vultr.php:1415]: Added check to prevent unnecessary API calls

**Review by tysonphillips — APPROVED** [review]

**Pull request merged**